experiment/test_config.yaml

```diff
@@ -690,7 +690,7 @@ nodeCommon:
     envs:
     - CLEANUP=true
     - GCE_ZONE=us-central1-f
-    - KUBELET_ARGS=--cgroups-per-qos=true --cgroup-root=/
+    - KUBELET_ARGS="--cgroups-per-qos=true --cgroup-root=/"
```

Comment:
If you will send follow up PRs to fix the TODO and you want to see the test result then I'm ok to merge this first :-)